resources/views/livewire/all-projects.blade.php

```diff
@@ -50,7 +50,10 @@
 
     <div class="stack with-sidebar with-sidebar:2/3">
         <div class="filters">
-            <h2 class="visually-hidden">{{ __('Filters') }}</h2>
+            <h3 class="mb-6 mt-0">{{ __('Filters') }}</h3>
```

Comment:
I checked the Figma design and it is h3 on Figma. I can change it back to h2 if you think that's more appropriate.

Reply:
I changed it back to h2.

Reply:
I believe it would make more sense to be an `<h2>`. In DOM order the page would go `<h1>` > `<h3>` > `<h2>` which can be confusing. Additionally if you look at the expanding/accordion elements contained within the filters section, they are marked up with `<h3>` elements already. Which would put the heading for the section at the same level as it's content.